tests/check/test_stream.c

```diff
@@ -59,6 +60,9 @@ static const uint8_t p_nalu_h264[DUMMY_NALU_SIZE] = {0x01, 0x00, 0xff, 0x00, 0x8
 static const uint8_t pps_nalu_h264[DUMMY_NALU_SIZE] = {0x28, 0x00, 0xff, 0x00, 0x80};
 static const uint8_t sei_nalu_h264[DUMMY_SEI_SIZE] = {0x06, 0x05, 0x12, 0xaa, 0xaa, 0xaa, 0xaa,
     0xaa, 0xaa, 0xaa, 0xaa, 0xaa, 0xaa, 0xaa, 0xaa, 0xaa, 0xaa, 0xaa, 0xaa, 0x00, 0x00, 0x80};
+static const uint8_t oms_sei_nalu_h264[DUMMY_SEI_SIZE] = {0x00, 0x5b, 0xc9, 0x3f, 0x2d, 0x71, 0x5e,
```

Comment:
This does still not follow the H.264 standard. See for example https://www.itu.int/rec/T-REC-H.264